test.ini

```diff
@@ -15,6 +15,7 @@ ckan.plugins = datagovtheme
 ckan.legacy_templates = no
 sqlalchemy.url = postgresql://ckan:pass@db/ckan
 solr_url = http://solr:8983/solr/ckan
+ckanext.geodatagov.bureau_csv.url_default = https://resources.data.gov/schemas/dcat-us/v1.1/omb_bureau_codes.csv
```

Comment:
What you've identified here is an external dependency. Let's remove it. I would commit the csv to the extension. Maybe it should live as data that has to be loaded at runtime, but for now let's hard code. It's not like that csv is changing with any frequency. 

Then you can use the resource manager api and you probably sidestep then conversion altogether.